extensions/aissemble-extensions-model-training-api-sagemaker/pyproject.toml

```diff
@@ -27,6 +27,8 @@ setuptools = "^69.0.3"
 kappa-maki = ">=1.0.1"
 pylint = "^3.1.0"
 
+[tool.pylint.'MESSAGES CONTROL']
+extension-pkg-whitelist = "pydantic"
```

Comment:
Just to make sure I understand, `BaseModel` is an import from Pydantic that is being flagged by Pylint because of some issue inherent to Pydantic, so we disable linting for Pydantic?

Reply:
Yes, linking the [issue](https://github.com/pydantic/pydantic/issues/1961) that has been flagged within the pydantic package.

Reply:
Ah might be nice to link that issue in the pyproject.toml file as well.

Reply:
Added, thanks Emily!